pacai/bin/capture.py

```diff
@@ -641,7 +640,6 @@ def readCommand(argv):
 
     # Choose a layout.
     if options.layout.startswith('RANDOM'):
-        seed = random.randint(0, 99999999)
         if (options.layout != 'RANDOM'):
             seed = int(options.layout[6:])
```

Comment:
Don't replace the other seed.
Although the previous value is not used again in this method, it makes it easier to get confused.
Just call this variable something else (like "layoutSeed" or something) and initialize it at the existing seed.